docs/appengine/taskqueue/push/defer.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package counter
+
+import (
+	"context"
+
+	"google.golang.org/appengine/delay"
+)
+
+func example() {
+	var ctx context.Context
+
+	// [START deferred_tasks]
+	var expensiveFunc = delay.Func("some-arbitrary-key", func(ctx context.Context, a string, b int) {
```

Comment:
Consider swapping `ctx` for `_` to avoid shadowing the top level `ctx` variable.

Reply:
I think this is OK to shadow? This defines a new function, then `ctx` is passed in when it's called. I left it outside the region tag because users probably don't want want to use `context.Background()` directly in prod.